Add Emeritus Maintaner role; Add 2 days period for merging PRs (#841)
serverlessworkflow/specification: GOVERNANCE.md

1) Providing guidance and mentorship to current maintainers and contributors.
2) Offering historical context and insights based on their past experiences.
3) Participating in discussions and reviews on an advisory basis, without the obligations of active maintainers.

## Adding maintainers

Maintainers are first and foremost contributors who have demonstrated their commitment to the long-term success of the project. Contributors wishing to become maintainers are expected to be deeply involved in contributing code, reviewing pull requests, and triaging issues for more than three months.

Just contributing does not make you a maintainer; it is about building trust with the current maintainers and being someone they can depend on to make decisions in the project's best interest.

Periodically, the existing maintainers curate a list of contributors who have shown regular activity over the prior months. From this list, maintainer candidates are selected and proposed on a pull request. Only one maintainer per organization is allowed to avoid conflicts of interest.

> After a candidate is announced on the pull request, the existing maintainers have fourteen business days to discuss, raise objections, and vote on the candidate. Votes can be cast on via pull request comments. Candidates must be approved by at least 66% of the current maintainers. The reviewer role has the same process but only requires 33% approval from current maintainers. Only maintainers of the repository the candidate is proposed for are allowed to vote.

Reviewer: ```suggestion
After a candidate is announced on the pull request, the existing maintainers have fourteen business days to discuss, raise objections, and vote on the candidate. Votes can be cast via pull request comments. Candidates must be approved by at least 66% of the current maintainers. The reviewer role has the same process but only requires 33% approval from current maintainers. Only maintainers of the repository the candidate is proposed for are allowed to vote.
```

> If approved, a maintainer will contact the candidate to invite them to open a pull request adding themselves to the MAINTAINERS file. The voting process can also take place inside a pull request if a maintainer has already discussed the candidacy with the candidate and is willing to sponsor them by opening the pull request. The candidate becomes a maintainer once the pull request is merged.

Reviewer: I think this paragraph can be removed since it's now a dup of the previous one. Just might want to add that the PR is for the MAINTAINERS file.

Main responsibilities of maintainers include:

1) Sharing responsibility for the project's success.
2) Making a long-term, recurring time investment to improve the project.
3) Performing necessary tasks, even if they are not the most interesting or fun.

## Reviewers

A reviewer is a core role within the project. They share in reviewing issues and pull requests.

> Their pull request approvals are needed to merge significant code changes into the project.

Reviewer: Might want to remove "significant" since don't all PRs require a review?

## Emeritus Maintainers

> Emeritus maintainers are retired maintainers who have chosen to remain involved in the project as advisors.

Reviewer: I wonder if it would be good to soften it a bit to include folks who can't remain involved (even as advisors) just as a way to say "[thanks for all the fish](https://ell.stackexchange.com/questions/115782/what-does-the-line-so-long-and-thanks-for-all-the-fish-mean)".

Perhaps:
Emeritus maintainers are retired maintainers who have provided valuable contributions to the project in the past but are not able to dedicate the time necessary to be fully active maintainers going forward. While their efforts will be focused elsewhere, it is hoped that they will try to find the time to continue to be active participants in the community by:
1) providing....

or something like that

Their main responsibilities include:

1) Providing guidance and mentorship to current maintainers and contributors.
2) Offering historical context and insights based on their past experiences.
3) Participating in discussions and reviews on an advisory basis, without the obligations of active maintainers.

## Adding maintainers

Maintainers are first and foremost contributors who have demonstrated their commitment to the long-term success of the project. Contributors wishing to become maintainers are expected to be deeply involved in contributing code, reviewing pull requests, and triaging issues for more than three months.

Just contributing does not make you a maintainer; it is about building trust with the current maintainers and being someone they can depend on to make decisions in the project's best interest.

> Periodically, the existing maintainers curate a list of contributors who have shown regular activity over the prior months. From this list, maintainer candidates are selected and proposed on the project mailing list. Only one maintainer per organization is allowed to avoid conflicts of interest.

Reviewer: do you reach out to the potential candidate prior to doing this to make sure they want to be a maintainer?

Author: Certainly!

After a candidate is announced on the project mailing list, the existing maintainers have fourteen business days to discuss, raise objections, and vote on the candidate. Votes can be cast on the mailing list or via pull request comments. Candidates must be approved by at least 66% of the current maintainers. The reviewer role has the same process but only requires 33% approval from current maintainers. Only maintainers of the repository the candidate is proposed for are allowed to vote.

> If approved, a maintainer will contact the candidate to invite them to open a pull request adding themselves to the MAINTAINERS file. The voting process can also take place inside a pull request if a maintainer has already discussed the candidacy with the candidate and is willing to sponsor them by opening the pull request. The candidate becomes a maintainer once the pull request is merged.

Reviewer: I like what's describe here better than the mailing list... just because it's faster. So,
1 - talk to candidate to make sure they want to be a maintainer
2 - open a PR to add them, and this kicks-off the 14 day review
3 - if approved, merge it

## Adding Emeritus Maintainers

> To transition a maintainer to an emeritus role, the current maintainers can nominate a retiring maintainer who has expressed interest in staying involved as an advisor. The nomination process follows the same voting and approval procedures as adding new maintainers, requiring a 66% approval vote from the current maintainers. Once approved, the emeritus maintainer is added to the EMERITUS file and announced to the community.

Reviewer: ```suggestion
To transition a maintainer to an emeritus role, the process follows the same voting and approval procedures as adding new maintainers, a new PR is created that requires a 66% approval vote from the current maintainers. Once approved, the emeritus maintainer is added to the EMERITUS file and announced to the community.
```

Reviewer: Minor thing, but did you want to wrap things at 80 columns to be consistent with the rest of the file
